browser/src/layer/tile/WriterTileLayer.js

```diff
@@ -120,11 +120,12 @@ L.WriterTileLayer = L.CanvasTileLayer.extend({
 		// 1s after the last invalidation, update the preview
 		clearTimeout(this._previewInvalidator);
 		this._previewInvalidator = setTimeout(L.bind(this._invalidatePreviews, this), this.options.previewInvalidationTimeout);
+		this._update();
```

Comment:
does that add more invalidations of canvas?

Reply:
I removed it in the other commits. That's right, it was unnecessary.